experimenter/experimenter/openidc/middleware.py

```diff
@@ -28,7 +28,10 @@ def __call__(self, request):
             pass
 
         default_email = settings.DEV_USER_EMAIL if settings.DEBUG else None
-        openidc_email = request.META.get(settings.OPENIDC_EMAIL_HEADER, default_email)
+        openidc_email = request.META.get(
+            settings.OPENIDC_EMAIL_HEADER,
+            request.META.get(f"HTTP_{settings.OPENIDC_EMAIL_HEADER}", default_email),
```

Comment:
Why this change in this patch?

Reply:
So something somewhere is rewriting headers to prepend `HTTP_` so now `HTTP_X_FORWARDED_USER` appears as `HTTP_HTTP_X_FORWARDED_USER` but only sometimes?  So I guess now we have to check for both 🤷‍♀️

Reply:
Actually yeah I'll add a comment there about it.  It might be the test client, it's not clear.

Reply:
We should also probably fix that, thats jank as hell.

Reply:
It is indeed.  Filed as a followup: https://github.com/mozilla/experimenter/issues/10318